addons/crm/report/crm_activity_report_views.xml

```diff
@@ -104,7 +104,7 @@
             name="Activities"
             parent="crm_menu_report"
             action="crm_activity_report_action"
-            sequence="3"/>
+            sequence="4"/>
```

Comment:
?

Reply:
Since Forecast is supposed to be the second menuitem of CRM>Reporting, I did increase every subsequent items